OpenRA.Mods.Common/Widgets/ButtonWidget.cs

```diff
@@ -249,7 +249,7 @@ public override void Draw()
 
 			var position = GetTextPosition(text, font, rb);
 
-			var hover = Ui.MouseOverWidget == this || Children.Any(c => c == Ui.MouseOverWidget);
+			var hover = Ui.MouseOverWidget == this || Children.Find(c => c == Ui.MouseOverWidget) != null;
```

Comment:
Is the idea that Find is implemented directly on List, so we win by not treating this as a general enumerable?

IMO this isn't immediately obvious, so it would be good if we could have a `// PERF: ...` comment here to explain why this is intentional.

Reply:
:+1: